packages/forms-web-app/src/validators/householder-planning/eligibility/date-decision-due-householder.js

```diff
@@ -0,0 +1,24 @@
+const { body } = require('express-validator');
+const { isAfter } = require('date-fns');
+const dateInputValidation = require('../../custom/date-input');
+
+const rules = () => [
+  ...(dateInputValidation('date-decision-due-householder', 'the Decision Date') || []),
```

Comment:
Is this the correct error message?

Reply:
Yes, Shahid has not edited the ticket yet to reflect GDS

Reply:
The test ix clearly wrong. Do you want me to chase Shahid or you want to chase him?